### DISCUSSION TITLE
Question about 3d PDE simulation

### DISCUSSION THREAD

**Discussion by qingpowuwu** [zwicker-group/py-pde#528] [General]
Great work! The documentation, implementation, and examples are really concise and clear, I like it. I was wondering if you could provide some demonstrations for simulating 3D partial differential equations? Thank you very much!"

**Comment by david-zwicker** [comment]
I unfortunately don't have time to provide extensive examples. If you have questions, I can try to answer them. Even better, if you want to contribute examples, I'm happy to look at pull requests.
